artemis/sql_messages_example.py

```diff
@@ -0,0 +1,588 @@
+SQL_ERROR_MESSAGES = [
```

Comment:
change the name of the module (sql_messages_example -> e.g. sql_injection_data_check)

Reply:
Done.